core/server/worker/src/main/java/alluxio/worker/netty/AbstractWriteHandler.java

```diff
@@ -85,10 +87,12 @@
    * EOF: the end of file.
    * CANCEL: the write request is cancelled by the client.
    * ABORT: a non-recoverable error is detected, abort this channel.
+   * FLUSH: flush the buffered data.
    */
   private static final ByteBuf EOF = Unpooled.buffer(0);
   private static final ByteBuf CANCEL = Unpooled.buffer(0);
   private static final ByteBuf ABORT = Unpooled.buffer(0);
+  private static final ByteBuf FLUSH = Unpooled.buffer(0);
```

Comment:
In the `release` method, we should not release this buffer.

Reply:
Added buffer != FLUSH check in the release method, thanks!